### PR TITLE
Upgrade to CDH5 and Java 7

### PR DESCRIPTION
**Pull request by stephanh**

### CODE REVIEW

**Comment by stephanh** [comment]
@SamRoberts @AndreasKostler can you please review.

**Comment by AndreasKostler** [comment]
Looks good to me. However, what about bumping the version of the assembly plugin in build.scala for sat versions 0.13.6+?

**Comment by stephanh** [comment]
Good idea.

**Comment by stephanh** [comment]
The latest version of assembly is `0.13.0`. It's a third party plugin.

File: uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala [hunk 1]
```diff
@@ -82,8 +83,7 @@ object UniformDependencyPlugin extends Plugin {
     )
 
     def hive(version: String = versions.hive) = Seq(
-      "org.apache.hive"          % "hive-exec"                      % version,
-      "org.apache.hive"          % "hive-builtins"                  % version
```
Comment (SamRoberts) [inline uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala]: is removing `hive-builtins` intentional?

**Comment by SamRoberts** [comment]
:+1:

File: uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala (hunk 1 above)
Comment (stephanh) [inline uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala]: Yes. `hive-builtins` doesn't exists for CDH5. I think it exists only for hive 0.10 and below.

**Comment by AndreasKostler** [comment]
:+1:

**Pull request merged**